src/http/multipart/builder.cr

```diff
@@ -25,6 +25,7 @@ module HTTP::Multipart
     # boundary parameter added.
     #
     # ```
+    # io = IO::Memory.new
     # builder = HTTP::Multipart::Builder.new(io, "a4VF")
     # builder.content_type("mixed") # => "multipart/mixed; boundary=\"a4VF\""
```

Comment:
Actually not sure about this one, I think the example is meant to show to pass any io and an empty one would actually be the extraordinary case.

Reply:
Basically I like the minimally closed executable code to use the function. Therefore, although I added code fragments as above, I agree that it is important not to misunderstand the essence. Should we remove it? or keep it with adding some comments?